### DISCUSSION TITLE
Can I log in to Web3Auth without my social login?

### DISCUSSION THREAD

**Discussion by shahbaz17** [Web3Auth/Web3Auth#764] [Help]
If I lose my social account but still have the remaining elements like device share and backup phrase, can I still login using web3auth?

**Comment by BboyStatix** [comment]
Hi there currently Web3Auth SDK does not provide any methods to login without social login. However, you can make use of the underlying technology underpinning Web3Auth, tKey to reconstruct your share. Here's a demo https://tech.tor.us/.
Here's the link https://web3auth.io/docs/sdk/self-host/ where you can see how you can implement this yourself.
